modules/monitoring-reviewing-monitoring-dashboards-developer.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * monitoring/reviewing-monitoring-dashboards.adoc
+
+[id="reviewing-monitoring-dashboards-developer_{context}"]
+= Reviewing monitoring dashboards as a developer
+
+In the Developer perspective, you can view dashboards relating to a selected project. You must have access to monitor a project to view dashboard information for it.
+
+.Prerequisites
+
+* You have access to the cluster as a developer or as a user with view permissions for the project that you are viewing the dashboard for.
+
+.Procedure
+
+. In the Developer perspective in the {product-title} web console, navigate to *Monitoring* -> *Dashboard*.
+
+. Choose a project in the *Project:* list.
+
+. Choose a workload in the *All Workloads* list.
+
+. Select a *Time Range*.
+
+. Select a *Refresh Interval*.
```

Comment:
I think this should this say `Optional`. Or maybe just remove this line since it should be self explanatory?

Reply:
Resolved.